pilot/pkg/model/proxy_config.go

```diff
@@ -145,6 +145,7 @@ func toMeshConfigProxyConfig(pc *v1beta1.ProxyConfig) *meshconfig.ProxyConfig {
 	if pc.Image != nil {
 		mcpc.Image = pc.Image
 	}
+	mcpc.PreserveCase = pc.PreserveCase
```

Comment:
Assuming we _do_ go with a field on ProxyConfig (as opposed to an annotation), then this makes sense. I think [this ](https://github.com/istio/api/pull/3350/files) would have to merge first though.